satpy/tests/reader_tests/test_satpy_cf_nc.py

```diff
@@ -33,6 +33,60 @@
 # - tmp_path
 
 
+def _create_test_netcdf(filename, resolution=742):
+    res = 2
```

Comment:
Maybe `size` would be a better name for this variable?
```suggestion
    size = 2
```

Reply:
Good point